### ISSUE TITLE
children and loader props should be PropTypes.node

### ISSUE THREAD

**Issue by hrafne** [CassetteRocks/react-infinite-scroller#74]
Wouldn't it be more fitting to validate `children` and `loader` prop with `PropTypes.node`, as in anything that can be rendered?

**Comment by danbovey** [comment]
Yep, I'm pretty sure that's left over from the original repository.

**Comment by rajiff** [comment]
I keep getting this warning 

`Warning: Failed prop type: The prop `children` is marked as required in `InfiniteScroll`, but its value is `undefined`.
`
How to resolve this? If i provide a empty object to children, they it opens so many errors, there is no documentation also on this

**Comment by danbovey** [comment]
Must admit I've never tested an empty state but I would have thought that even `items.map` when `items.length` is `0` is a valid child.

```
<InfiniteScroll>
    // Items here are the children 'prop', it can not be empty or undefined
</InfiniteScroll>
```

[A full example to pass children to the component can be found here](https://github.com/CassetteRocks/react-infinite-scroller/blob/master/docs/src/index.js#L67)

**Comment by danbovey** [comment]
Will get round to changing this soon as potentially people could be using something other than `items.map` etc.

**Comment by danbovey** [comment]
Fixed in #155
